python/tvm/tir/schedule/schedule.py

```diff
@@ -2092,9 +2092,9 @@ def before_tensorize(
 
             @T.prim_func
             def mma_desc(a: T.handle, b: T.handle, c: T.handle) -> None:
-                A = T.match_buffer(a, (16, 16), align=128, offset_factor=1)
-                B = T.match_buffer(b, (16, 16), align=128, offset_factor=1)
-                C = T.match_buffer(c, (16, 16), align=128, offset_factor=1)
+                A = T.match_buffer(a, (16, 16), align=64, offset_factor=1)
+                B = T.match_buffer(b, (16, 16), align=64, offset_factor=1)
+                C = T.match_buffer(c, (16, 16), align=64, offset_factor=1)
```

Comment:
mma does use `align=128`, so let's revert this change here :-)

Reply:
Ah make sense, reverted. Is there anywhere else we'd want 128 byte alignment? I just did a global search and replace.

Reply:
not super familiar with non-cuda cases. probably @Hzfengsy @spectrometerHBH could help!

Reply:
Not sure about other backends. But it's fine if we can pass the tests